### ISSUE TITLE
Bug: MonitorSleep command turns off monitor only for a second or two

### ISSUE THREAD

**Issue by knezmilos13** [hass-agent/HASS.Agent#108]
Using MonitorSleep command turns off monitor only for a second or two, then it comes back on. It worked properly when I was using the original HASS.Agent; I have the original still installed on my work laptop and there the MonitorSleep command works properly.

**To Reproduce**
Trigger MonitorSleep manually or through an automation.

**Expected behavior**
Monitor turns off and stays off

**Misc info (please complete the following information):**
 - Windows build (ideally screenshot/info of `winver.exe` output): 
![image](https://github.com/hass-agent/HASS.Agent/assets/2862958/4b0ffb8e-0838-4991-9a09-bc93bbc2eda5)
 - Windows' UI language: English
 - HASS.Agent version: 2.1.0-beta1

 - [x] Installed via installer
 - [ ] Installed manually
 - [x] Problem occurs in HASS.Agent
 - [ ] Problem occurs in Satellite Service

**Comment by amadeo-alex** [comment]
Hello,

> It worked properly when I was using the original HASS.Agent
On the same machine that has now issues? Is it working currently when you use original HASS.Agent on the same machine?

**Comment by knezmilos13** [comment]
Ok, update, after I wrote this issue, it worked correctly once, then not worked again, and has now worked multiple times as-is (I haven't installed the original HASS agent to check). I'm thinking that it was maybe just a coincidence that it started acting up when I switched from the original to this version. My prime suspect is now my gamepad - it's always connected to my PC and I caught it sending some inputs in a game I played using keyboard + mouse. Using the gamepad thumbstick while in Windows doesn't acutally do anything, but it does turn the sleeping screen back on. This would explain why my work laptop wasn't acting up - because the gamepad is not connected to it.
